### ISSUE TITLE
Rating Page 

### ISSUE THREAD

**Issue by daksh1210jain** [Spandan-Bhattacharya/Solve_it#92]
Adding rating page using HTML CSS

**Comment by daksh1210jain** [comment]
Kindly assign me this issue

**Comment by Spandan-Bhattacharya** [comment]
Can you please elaborate what you want to implement?

**Comment by daksh1210jain** [comment]
Sir basically a page which will enable user to give rating out of 5 stars and also a feedback

**Comment by daksh1210jain** [comment]
under JWOC

**Comment by Spandan-Bhattacharya** [comment]
Okay but for this backend will backend be required?

**Comment by daksh1210jain** [comment]
yes sir it might be required but if we make it without backend then also it can be made just inorder to get customer satisfaction

**Comment by Spandan-Bhattacharya** [comment]
Yes but without backend, getting the user feedback would be very difficult.
So currently i think this issue is not valid for the project
